lib/rubocop/cop/style/word_array.rb

```diff
@@ -11,7 +11,7 @@ module Style
       #
       # Configuration option: MinSize
       # If set, arrays with fewer elements than this value will not trigger the
-      # cop. For example, a `MinSize of `3` will not enforce a style on an array
+      # cop. For example, a `MinSize` of `3` will not enforce a style on an array
```

Comment:
Sorry, there is another cause of CI failure.
This line is too long.

https://ci.appveyor.com/project/bbatsov/rubocop/build/952/job/qfvngr1ko9069oid

```
Offenses:
lib/rubocop/cop/style/word_array.rb:14:81: C: Line is too long. [81/80]
      # cop. For example, a `MinSize` of `3` will not enforce a style on an array
                                                                                ^
1009 files inspected, 1 offense detected

```

Reply:
Thanks! I moved "array" to the next line and regenerated the docs with the `rake` task, let's hope this is fixed now 😄